spec/helpers/alert_notifier_helper_spec.rb

```diff
@@ -147,4 +147,52 @@
       end
     end
   end
+
+  describe "#subject" do
+    let(:alert) { create(:alert, address: "123 Sample St") }
+    let(:application) do
+      mock_model(Application, address: "Bar Street",
+      description: "Alterations & additions", council_reference: "007",
```

Comment:
I think this multi line method call would be clearer with the parameters indented. [From the Ruby style guide](https://github.com/bbatsov/ruby-style-guide#no-double-indent):

>  Align the parameters of a method call if they span more than one line. When aligning parameters is not appropriate due to line-length constraints, single indent for the lines after the first is also acceptable.